src/Microsoft.AspNet.Mvc.Core/ActionResults/FileContentResult.cs

```diff
@@ -19,13 +20,14 @@ public class FileContentResult : FileResult
 
         /// <summary>
         /// Creates a new <see cref="FileContentResult"/> instance with
-        /// the provided <paramref name="fileContents"/>.
+        /// the provided <paramref name="fileContents"/> and the
+        /// provided <paramref name="contentType"/>.
         /// </summary>
         /// <param name="fileContents">The bytes that represent the file contents.</param>
-        public FileContentResult([NotNull] byte[] fileContents)
-            : base(contentType: null)
```

Comment:
If we compare to MVC5, content type was always required a content-type:
http://aspnetwebstack.codeplex.com/SourceControl/latest#src/System.Web.Mvc/FileResult.cs